libcxx/test/libcxx/gdb/gdb_pretty_printer_test.sh.cpp

```diff
@@ -12,7 +12,7 @@
 // UNSUPPORTED: c++03
 
 // TODO: Investigate these failures which break the CI.
-// UNSUPPORTED: clang-18, clang-19, clang-20
+// UNSUPPORTED: clang-18, clang-19, clang-20, clang-21
```

Comment:
Let's revive the patch to remove this. It's been broken for 1.5 years. If nobody cares about it, we should get rid of it.

Reply:
It's not been broken. Our CI is. This actually works for every clang version listed here, except in the CI.

Reply:
Do we know why it does not work in our CI?

Reply:
IIRC the last time it mostly worked except in the optimized build. I've disabled it for everything, since it was already the same day as the LLVM 20 branch and the test was already disabled for all the other versions. We should look into it and maybe just disable it for the optimized build.